.upstream-tests/utils/libcudacxx/test/config.py

```diff
@@ -748,8 +751,9 @@ def configure_compile_flags_header_includes(self):
                                  and self.cxx_stdlib_under_test != 'libc++'):
             self.lit_config.note('using the system cxx headers')
             return
-        if self.cxx.type != 'nvcc' and self.cxx.type != 'pgi':
-            self.cxx.compile_flags += ['-nostdinc++']
+        # I don't think this is required, since removing it helps clang-cuda compile and libcudacxx only supports building in CUDA modes?
+        # if self.cxx.type != 'nvcc' and self.cxx.type != 'pgi':
+        #    self.cxx.compile_flags += ['-nostdinc++']
```

Comment:
Yeah I think you are right. Change the comment to state this as a fact, with a note to reenable it if we ever decide to start actually testing in a C++ mode.